core/api/views/country_programme.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import mixins, generics
+from core.api.filters.country_programme import CountryProgrammeReportFilter
+
+from core.api.serializers import CountryProgrammeReportSerializer
+from core.models.country_programme import CountryProgrammeReport
+
+
+# view for country programme records
+class CountryProgrammeReportListAPIView(mixins.ListModelMixin, generics.GenericAPIView):
```

Comment:
In principle we should add some more filtering based on the current user and their roles (a party user should not be allowed to see other countries' reports; Only secretariat/admin users are allowed to see all reports). 
We can do this later, though.

Reply:
I think we should do this when we add user roles. In that task we should add all the user permissions and update the endpoints